deploy/kiali/kiali_cr.yaml

```diff
@@ -623,6 +623,10 @@ spec:
 #      custom_headers: {}
 #      health_check_url: ""
 #      is_core: true
+#      thanos_proxy:
+#        enabled: false
+#        retention_period: "7d"
+#        scrape_interval: "30s"
```

Comment:
you should explain these settings above -- around line 607. Just add a comment up there to briefly talk about these so people know what they are for.

Reply:
Yes you're right, let me do it